src/servers/src/otlp/metrics.rs

```diff
@@ -38,33 +37,7 @@ const INSTANCE_KEY: &str = "instance";
 
 const UNDERSCORE: &str = "_";
 
-// see: https://prometheus.io/docs/guides/opentelemetry/#promoting-resource-attributes
-// instance and job alias to service.instance.id and service.name that we need to keep
-const DEFAULT_ATTRS: [&str; 19] = [
-    "service.instance.id",
-    "service.name",
-    "service.namespace",
-    "service.version",
-    "cloud.availability_zone",
-    "cloud.region",
-    "container.name",
-    "deployment.environment",
-    "deployment.environment.name",
-    "k8s.cluster.name",
-    "k8s.container.name",
-    "k8s.cronjob.name",
-    "k8s.daemonset.name",
-    "k8s.deployment.name",
-    "k8s.job.name",
-    "k8s.namespace.name",
-    "k8s.pod.name",
-    "k8s.replicaset.name",
-    "k8s.statefulset.name",
-];
```

Comment:
Wait, we still need this list as default promote list.

Reply:
We will always promote these items, until there is a request. When user specify the promote resource attirbute lists, we promote those in additional to these.